package.json

```diff
@@ -233,7 +234,7 @@
     "react-intl": "2.7.2",
     "react-lottie": "1.2.3",
     "react-markdown": "4.3.1",
-    "react-polymorph": "0.9.8-rc.11",
+    "react-polymorph": "https://github.com/input-output-hk/react-polymorph.git#ca76f01a202f691caaf9b9c1b59ccf0fbe8be91a",
```

Comment:
Does it work to reference commits with Daedalus builds nowadays?

Reply:
No. This is only temporary. https://github.com/input-output-hk/daedalus/pull/2512#issuecomment-823031379

Reply:
I will make a new RP release for this PR